doc/help/commands.asciidoc

```diff
@@ -918,6 +918,9 @@ Repeat a given command.
 * +'times'+: How many times to repeat.
 * +'command'+: The command to run, with optional args.
 
+==== count
+Multiplies with 'times' when given
+
```

Comment:
This looks like you documented it here by hand? Instead, document it in the docstring of `repeat` and then run the `src2asciidoc.py` script (see the earlier comment from @jgkamat).